src/main/resources/add-ons-to-index.json

```diff
@@ -2593,10 +2621,10 @@
       "uid": "org.openmrs.module.reporting",
       "type": "OMOD",
       "name": "Reporting",
+      "icon": "bar-chart",
       "tags": [
         "reporting"
       ],
-      "icon": "bar-chart",
```

Comment:
I just swapped this to follow the convention as described [here](https://github.com/openmrs/openmrs-contrib-addonindex/blob/da43a6c5f44eadad8c96866f3f193fe38655e758/src/main/resources/add-ons-to-index.json#L1244).